content/en/providers/cloud-compute/openstack/aai.md

```diff
@@ -669,7 +669,7 @@ and Demo environments already using Keycloak since June 24th 2022 and Production
 environment expected to migrate during July 2022.
 
 A general guide on migration is available on
-[Check-in documentation](../../../check-in/sp/#client-migration-to-keycloak), in
+[Check-in documentation](../../check-in/sp/#client-migration-to-keycloak), in
```

Comment:
Could you please revert this change? The link in the [preview](https://docs.egi.eu/documentation/469/providers/cloud-compute/openstack/aai/#client-migration-to-keycloak) is broken.
Here is a screenshot of the redirect when I hover the mouse above the `Check-in documentation` link:
![Screen Shot 2022-07-06 at 12 13 38](https://user-images.githubusercontent.com/29930145/177516321-ab2704fd-7495-46dd-94b0-265b7814ed6c.png)
CC: @gwarf

Reply:
I more or less reverted the change, seems it's again a false error, not exactly sure what is happening here :/

Reply:
Let's ignore the error and bypass it for the merge